angular/projects/spark-angular/src/lib/components/inputs/sprk-checkbox.stories.ts

```diff
@@ -602,23 +596,19 @@ export const hugeHelperText = () => ({
   `,
 });
 
-hugeHelperText.story = {
-  name: 'Huge Helper Text',
-  parameters: {
-    jest: [
-      'sprk-checkbox-input.directive',
-      'sprk-checkbox-label.directive',
-      'sprk-fieldset.directive',
-      'sprk-legend.directive',
-      'sprk-label.directive',
-      'sprk-checkbox-item.component',
-      'sprk-checkbox-group.component',
-    ],
-  },
+huge.hugeHelperText = {
```

Comment:
Should this be `hugeHelperText.parameters` ?

Reply:
lol yup it sure should be haha